assets/js/quiz.js

```diff
@@ -0,0 +1,38 @@
+var wrongA
+var rightA
+var shuffled
+var quizOver = false
+
+
+function displayQuestion() {
+  if (quizOver = false) {
```

Comment:
You need to do a strict comparison with 3 === 
`quizOver = false`

Reply:
updated to ===